### PR TITLE
Bump dependencies.

### PR DESCRIPTION
**Pull request by bernaferrari**
Sequel / Fix https://github.com/google/process.dart/pull/75.
I added the minor changelog version and kept the author as co-author.
cc @gspencergoog.

### CODE REVIEW

**Comment by bernaferrari** [comment]
cc @gspencergoog

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

**Comment by bernaferrari** [comment]
Thanks. Feel free to merge, I can't autosubmit here.. lol

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#### 5.0.1
```
Comment (jakemac53) [inline CHANGELOG.md:1]: It looks like 5.0.0 was never published, https://pub.dev/packages/process/versions.

Lets move this back to the 5.0.0 section and then release 5.0.0

**Pull request merged**